codebase-pipelines/custom_pipeline_update/update_pipeline.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+import boto3
+from typing import List
+
+
+def update_pipeline_stage_failure(pipelines: List[str]):
+    for pipeline_name in pipelines:
+        print(f"Updating pipeline {pipeline_name}")
+
+        client = boto3.client("codepipeline")
+
+        response = client.get_pipeline(name=pipeline_name)
+        pipeline = response["pipeline"]
+
+        stage_found = False
+        for stage in pipeline["stages"]:
+            if "Deploy" in stage["name"]:
```

Comment:
I wonder if this should be:
```suggestion
            if "Deploy" == stage["name"]:
```
to narrow down possible matches. Like is someone in the future adds a "Notify Deploy Status" stage or something.

Reply:
I think the stage name has the environment in it e.g. Deploy-dev